packages/edit-site/src/components/sidebar/index.js

```diff
@@ -25,11 +26,11 @@ function Sidebar() {
 				scope="core/edit-site"
 				complementaryAreaIdentifier="edit-site/global-styles"
 				title={ __( 'Global Styles' ) }
-				icon={ pencil }
+				icon={ typography }
 			>
-				<p>Global Styles area</p>
+				<GlobalStylesControls bubblesVirtually />
```

Comment:
Let's render this one before the "settings" tool.

Reply:
done at https://github.com/WordPress/gutenberg/pull/20530/commits/f1ba4d88892b37b18483f99ef29be745bff3fde2